modular_skyrat/modules/rocket-launcher/rocket-launcher.dm

```diff
@@ -0,0 +1,203 @@
+/obj/item/storage/belt/military/rocket
+	name = "rocket storage rig"
+	desc = "A storage rig configured to hold up to seven of your most valued rockets!"
+	icon_state = "explorer2"
+	worn_icon_state = "explorer2"
+
+/obj/item/storage/belt/military/rocket/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_items = 7
+	STR.display_numerical_stacking = TRUE
+	STR.max_combined_w_class = 30
+	STR.max_w_class = WEIGHT_CLASS_BULKY
+	STR.set_holdable(list(
+		/obj/item/ammo_casing/caseless/rocket
+		))
+
+/obj/item/storage/belt/military/rocket/full/PopulateContents()
+	generate_items_inside(list(
+		/obj/item/ammo_casing/caseless/rocket/smoke = 2,
+		/obj/item/ammo_casing/caseless/rocket/sabot = 2,
+		/obj/item/ammo_casing/caseless/rocket/smoke/phosphor = 1,
+		/obj/item/ammo_casing/caseless/rocket/smoke/sleeping = 1,
+		/obj/item/ammo_casing/caseless/rocket/emp = 1,
+	), src)
+
+/obj/item/ammo_casing/caseless/rocket/sabot
+	name = "\improper APCBCFSDSGLATM"
+	desc = "Why anyone would name a rocket 'Armour Piercing Capped Ballistic Capped Fin Stabilized Discarding Sabot Gun Launched Anti Tank Missile' is beyond your understanding."
+	icon = 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-ammo.dmi'
```

Comment:
`modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:71:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketlaunchersec.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:72:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-right.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:73:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-left.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:30:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-ammo.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:38:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-ammo.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:46:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-ammo.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:54:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-ammo.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:61:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-ammo.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:83:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-project.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:101:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-project.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:114:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-project.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:131:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-project.dmi': cannot find file
modular_skyrat/modules/rocket-launcher/rocket-launcher.dm:145:error: 'modular_skyrat/modules/rocket-launcher/icon/rocketsec-project.dmi': cannot find file`

Reply:
Yeah, I know, I know.
I didn't notice that my Github desktop didn't mark em correctly.
It all complied and worked fine when I tested it locally.
Just need to push the right stuff here.